core/model/configuration.py

```diff
@@ -1132,6 +1132,17 @@ def __get__(
 
         if setting_value is None:
             setting_value = self.default
+        elif self.type == ConfigurationAttributeType.NUMBER:
+            # In case we're a number, the default should replace any incorrect formatting
```

Comment:
Maybe best to remove this comment, since and exception might be raised here, instead of going with the default.

Reply:
Removed